app/templates/_gulpfile.js

```diff
@@ -98,6 +98,25 @@ gulp.task('plugins', function () {
 
 });
 
+gulp.task('modernizr' function () {
+
+  return gulp.src(source.scripts)
+    .pipe($.modernizr({
+      options: [
+        'setClasses'
+      ],
+      tests: [
+        'flexbox',
+        'flexboxlegacy',
+        'flexboxtweener'
+      ],
+      crawl: false
+    }))
+    .pipe($.uglify())
+    .pipe(gulp.dest(assets.scripts));
+
+})
```

Comment:
Expected an identifier and instead saw ')'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.